yt/funcs.py

```diff
@@ -1273,3 +1273,54 @@ def __missing__(self, key):
             return val
 
     return DictWithFactory
+
+
+def levenshtein(seq1, seq2, max_dist):
```

Comment:
I want to trust the implementation is correct here, can you add unit tests ?

Reply:
Done!